frontend-react/src/components/Grid.tsx

```diff
@@ -0,0 +1,10 @@
+import classNames from "classnames";
+
+export interface GridRowProps {
+    children: React.ReactNode;
+    className?: string;
+}
+
+export function GridRow({ children, className }: GridRowProps) {
+    return <div className={classNames("grid-row", className)}>{children}</div>;
```

Comment:
Isn't this functionality already provided by [the Grid component](https://github.com/trussworks/react-uswds/blob/main/src/components/grid/Grid/Grid.tsx#L61)?

```tsx
<Grid className={ className } row>
    { children }
</Grid>
```

Reply:
Could be! If so, will drop this in favor of it. Thanks for the heads up!